assets/main/app.js

```diff
@@ -0,0 +1,105 @@
+String.prototype.hasVowels = function() {
+  var vowels = /[aeiou]/i;
+  return vowels.test(this);
+};
+
+String.prototype.toUpper = function() {
+  var upperCased = '';
+  for (var index = 0; index < this.length; index++) {
+    var currentValue = this.charCodeAt(index);
+    if (currentValue > 96 && currentValue < 123) {
+      upperCased += String.fromCharCode(currentValue & ~(1 << 5));
+    } else {
+      upperCased += this[index];
+    }
+  }
+  return upperCased;
+};
+
+String.prototype.toLower = function() {
+  var lowerCased = '';
+  for (var index = 0; index < this.length; index++) {
+    var currentValue = this.charCodeAt(index);
+    if (currentValue > 64 && currentValue < 91) {
+      lowerCased += String.fromCharCode(currentValue + 32);
+    } else {
```

Comment:
See if you can refactor this if-else statement so that you get rid of the **else** statement. Consider having the first if statement and use the **continue** keyword inside the if statement.